### PR TITLE
sys-cluster/pconsole: fix/use HTTPs, EAPI7, improve ebuild

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This updates the ebuild for sys-cluster/pconsole.
Please review.

diff -u:
```
--- pconsole-1.0-r3.ebuild      2018-07-16 19:10:19.485146052 +0200
+++ pconsole-1.0-r4.ebuild      2018-07-16 19:10:19.584143568 +0200
@@ -1,35 +1,29 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=2
+EAPI=7
 
-inherit eutils toolchain-funcs
+inherit toolchain-funcs
 
 DESCRIPTION="Tool for managing multiple xterms simultaneously"
 HOMEPAGE="https://github.com/walterdejong/pconsole"
 SRC_URI="http://www.xs4all.nl/~walterj/pconsole/${P}.tar.gz"
+
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~x86"
-IUSE=""
-
-DEPEND=""
-RDEPEND="virtual/ssh"
 
-src_prepare() {
-       epatch "${FILESDIR}"/${P}-exit-warn.patch
-}
+PATCHES=( "${FILESDIR}"/${P}-exit-warn.patch )
 
 src_compile() {
        emake LFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS}" \
-               CC="$(tc-getCC)" || die
+               CC="$(tc-getCC)"
 }
 
 src_install() {
-       dobin pconsole || die
-       fperms 4110 /usr/bin/pconsole || die
-       dodoc ChangeLog README.pconsole || die
-       dohtml public_html/pconsole.html || die
+       dobin pconsole
+       fperms 4110 /usr/bin/pconsole
+       dodoc ChangeLog README.pconsole public_html/pconsole.html
 }
 
 pkg_postinst() {
```

### CODE REVIEW

**Comment by mm1ke** [comment]
Ping, just rebased this PR, any chance for a merge?

File: sys-cluster/pconsole/pconsole-1.0-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Tool for managing multiple xterms simultaneously"
+HOMEPAGE="https://github.com/walterdejong/pconsole"
```
[the remark this answers is not in the document]
Comment (mm1ke) [reply sys-cluster/pconsole/pconsole-1.0-r4.ebuild]: thx, added the upstream information to the metadata.xml

File: sys-cluster/pconsole/pconsole-1.0-r4.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	dobin pconsole
+	fperms 4110 /usr/bin/pconsole
+	dodoc ChangeLog README.pconsole public_html/pconsole.html
+}
+
+pkg_postinst() {
+	echo
+	ewarn "Warning:"
+	ewarn "pconsole installed with suid root!"
+	echo
```
[the remark this answers is not in the document]
Comment (mm1ke) [reply sys-cluster/pconsole/pconsole-1.0-r4.ebuild]: thx, i've removed the echo's from the ebuild

**Comment by bircoph** [comment]
Thanks. Commits are cherry-picked upstream now. I'm not sure while github bot can't see them.

**Pull request closed without merging**